libs/dal/src/repositories/message/message.repository.ts

```diff
@@ -133,11 +207,7 @@ export class MessageRepository extends BaseRepository<MessageDBModel, MessageEnt
       };
     }
 
-    const updateQuery: Partial<MessageEntity> & EnforceEnvId = {
-      _subscriberId: subscriberId,
-      _environmentId: environmentId,
-      [markAs]: false,
-    };
+    const updateQuery = this.getReadSeenUpdateQuery(subscriberId, environmentId, markAs);
```

Comment:
The idea is to encapsulate all query object creation logic in one function, with your suggestion we have to maintain additionally these if statements. I don't feel it's more intuitive, because now you will have it spread around a few places.

Reply:
Probably the name of the method was not intuitive to me at first but not a blocker to me at all.